packages/react-vapor/src/components/checkbox/examples/CheckboxExamples.1.use.md

```diff
@@ -0,0 +1,31 @@
+## Guidelines
+
+(See also: **Guidelines for All Form Controls** - _coming soon!_)
```

Comment:
What's the point of referring to a section that does not yet exist? We will need to edit this string anyway when the section will become available. I would simply not mention it yet since its not yet available.